docs/geode-native-docs/serialization/data-serialization.html.md.erb

```diff
@@ -27,3 +27,249 @@ the best combination of versatility and ease-of-use for most applications.
 
 To learn more about other serialization options, see the [Data Serialization section in the _<%=vars.product_name_long%> User Guide_](serverman/developing/data_serialization/chapter_overview.html).
 
+# PdxSerializable Examples
+
+PdxSerializable provides custom serialization to an individual class. Fields within an object can be serialized separately from the rest of the class.
+
+The type of serialization and its implementation can be specified entirely in the client
+application, with no need to create corresponding code on the <%=product_name%> server.
+
+The native client release contains examples, written for .NET and C++, showing how a client application
+can register for serialization of custom objects using the .NET IPdxSerializable interface or the C++ PdxSerializable interface.
+
+The examples are located in `examples\dotnet\pdxserializable` and `examples/cpp/pdxserializable`, respectively. 
+
+## Setup
+
+PdxSerializable provides custom serialization to an individual class. Fields within an object can be serialized separately from the rest of the class.
+
+Setting up a class for PdxSerializable treatment requires some preparatory steps:
+
+- The class must inherit from the .NET IPdxSerializable interface or the C++ PdxSerializable interface.
+- You must provide serialization instructions for objects of this class. Specifically, you must implement the `ToData()` and `FromData()` methods for .NET applications, or the `toData()` and `fromData()` methods in C++ applications.
+- The class must be registered with the cache, which takes care of informing the server.
+
+The examples define the serializable class, Orders, and supplies its serialization and deserialization methods.
+Once these pieces are in place, execution is simple: the main.cpp routine registers the serializable class then performs some put and get operations.
+
+## Execution
+
+Both examples perform the same sequence of operations, displaying simple log entries as they run.
+
+- To run an example, follow the instructions in the README.md file in the example directory.
+- Review the source code in the example directory to see exactly how it operates.
+
+- Begin by running a script that sets up the server-side environment by invoking `gfsh` commands to create a region.
```

Comment:
Ultimately, the startserver.[sh, ps1] scripts create a region, but the also start a locator and server. Not sure if it is worth extending this statement, though.

Reply:
Added locator and server to the list.